app/grpc/gasestimation/gas_estimator.go

```diff
@@ -122,14 +128,29 @@ func (s *gasEstimatorServer) estimateGasPrice(ctx context.Context, priority TxPr
 	if err != nil {
 		return 0, err
 	}
+
+	// Get the network minimum gas price from the min fee module
+	minGasPrice, err := s.minGasPriceFn()
+	if err != nil {
+		// If we can't get the network min gas price, fall back to default
+		minGasPrice = appconsts.DefaultMinGasPrice
```

Comment:
not too sure if we should just return an error to the user here instead of using the DefaultMinGasPrice as it might be the wrong value